tools/rna_tools/rnaformer/macros.xml

```diff
@@ -0,0 +1,47 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="1.1.0">rnaformer</requirement>
+            <requirement type="package" version="2.1.2">pytorch</requirement>
```

Comment:
Why are all those dependencies not part of the rnaformer conda package?

Reply:
You're right I overlooked that, thanks for pointing it out. I added these as an update for the conda package.